src/h2_stream.erl

```diff
@@ -363,6 +363,14 @@ open({send_frame,
                 open
         end,
     {next_state, NextState, Stream};
+open(
+  {send_h, Headers},
+  #stream_state{}=Stream) ->
+    {next_state,
+     open,
```

Comment:
in my case I'm sending an error message, so I want to send an `RST_STREAM`. But that's a seperate issue perhaps